tasks/apt.yml

```
@@ -40,12 +50,13 @@
     - start redis
   ignore_errors: true
   register: use_redis5
-  when: openwisp2_redis_install
+  when: openwisp2_redis_install and use_redis6 is failed
 
 - name: Install redis 4 (fallback)
-  when: openwisp2_redis_install and use_redis5.failed
+  when: openwisp2_redis_install and (use_redis5 is failed or use_redis6 is failed)
```

In the current state this task will still execute if use_redis6 failed but use_redis5 was successful. 

It has happened in the build

https://github.com/openwisp/ansible-openwisp2/pull/299/checks?check_run_id=3723105424#step:7:270

It should be any one from below. 

```suggestion
  when: openwisp2_redis_install and (use_redis5 is failed and use_redis6 is failed)
```


use_redis5 will only get registered with redis6 failed. 
```suggestion
  when: openwisp2_redis_install and use_redis5 is failed
```

Also, this task should not ignore error.